### PR TITLE
Stories/inter coc updates

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/controllers/warehouse_reports/overlapping_coc_utilization_controller.rb [hunk 1]
```diff
@@ -84,6 +84,24 @@ def details
...
+        expires_in: CACHE_LIFETIME,
+      ) do
+        @report.limited_details_hash
+      end
+      layout = if request.xhr?
+        'content_only'
+      else
+        'application'
+      end
+      render clients_warehouse_reports_overlapping_coc_utilization_index_path, layout: layout
```
Comment (awisema) [inline app/controllers/warehouse_reports/overlapping_coc_utilization_controller.rb:100]: I think I'd prefer this to look like:
```
if request.xhr?
  render clients_warehouse_reports_overlapping_coc_utilization_index_path, layout: :content_only
else
  render clients_warehouse_reports_overlapping_coc_utilization_index_path
end
```

File: app/models/warehouse_report/overlapping_coc_by_project_type.rb [hunk 2]
```diff
@@ -177,24 +177,55 @@ def details_clients
     client_num = 0
     clients_by_id = shared_clients.index_by(&:id)
     relevant_client_ids = @non_overlapping ? non_overlapping_client_ids : overlapping_client_ids
```
Comment (awisema) [inline app/models/warehouse_report/overlapping_coc_by_project_type.rb:179]: Don't we usually prefer `relevant_client_ids = if ...` elsewhere? I don't mind `?:`, I just don't usually see it in this code...

Comment (eanders) [reply app/models/warehouse_report/overlapping_coc_by_project_type.rb:179]: We do, that is old code I didn't cleanup yet.

File: app/models/warehouse_report/overlapping_coc_by_project_type.rb [hunk 3]
```diff
@@ -177,24 +177,55 @@ def details_clients
...
+      start_date: start_date.iso8601,
+      end_date: end_date.iso8601,
+      cocs: [coc1, coc2].map do |coc|
+        { code: coc.cocnum, name: coc.cocname }
+      end,
+      clients: limited_details_clients,
+    }
+  end
+
+  def limited_details_clients
```
Comment (awisema) [inline app/models/warehouse_report/overlapping_coc_by_project_type.rb:205]: Was confused that 'limited details' has more data than the 'load` above, is this limited in the sense of the max 50?

Comment (eanders) [reply app/models/warehouse_report/overlapping_coc_by_project_type.rb:205]: You got it!

**Pull request merged**